hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+This release makes some internal changes to how shrinking is structure. It is
```

Comment:
Nit: "shrinking is structure[d]", 'mostly a pure refactoring' is vague.  I'd rephrase this along the lines of

> This release makes some internal changes which facilitate further improvements to the shrinking engine.  You may see some changes in the final shrunk examples, but this is unlikely to be significant.